mcs/provider.go

```diff
@@ -71,6 +72,22 @@ func newConfig(d *schema.ResourceData, terraformVersion string) (configer, error
 		},
 	}
 
+	if config.TenantID == "" {
+		config.TenantID = os.Getenv("OS_PROJECT_ID")
+	}
+	if config.UserDomainID == "" {
+		config.UserDomainID = os.Getenv("OS_USER_DOMAIN_ID")
+	}
+	if config.Password == "" {
+		config.Password = os.Getenv("OS_PASSWORD")
```

Comment:
do we refuse to use env variables without 'OS' prefix?

Reply:
since mcs use openrc files with OS_ prefix it still has sence